backend/src/components/institute/institute.js

```diff
@@ -416,6 +416,40 @@ async function updateSchoolContact(req, res) {
   }
 }
 
+async function deleteSchoolContact(req, res) {
+  try {
+    const token = getBackendToken(req);
+    const formatter = DateTimeFormatter.ofPattern('yyyy-MM-dd\'T\'HH:mm:ss');
+
+    const contact =  await utils.getData(token, `${config.get('server:institute:instituteSchoolURL')}/${req.params.schoolId}/contact/${req.params.contactId}`);
+
+    if (!contact) {
+      return res.status(HttpStatus.NOT_FOUND).json({
+        message: 'Contact not found'
+      });
+    }
+
+    let school = cacheService.getSchoolBySchoolID(req.params.schoolId);
```

Comment:
Minor but please move this snippet up above where you pull the contact on line 424.